README.md

```diff
@@ -4,6 +4,31 @@
 
 This code serves as a template for the creation of further building blocks with the purpose of giving all blocks a uniform structure and usage.
 
+## Prerequisites
+
+Install the following prereuisites:
+-   [docker](https://docs.docker.com/install/)
+-   [dobi](https://github.com/dnephin/dobi)
+
+## Howto use bb-buildingblock template
+
+Create a new repository and merge bb-buildingblock, e.g.:
+```bash
+git init myrepo
+cd myrepo
+git remote add bb-buildingblock https://github.com/elbb/bb-buildingblock.git
+git fetch -n bb-buildingblock
+git merge bb-buildingblock master
+```
+
+Adapt your own ci/cd pipeline. Use the pipeline from `example/ci` directory and adapt it. E.g.:
+
+```bash
+cp -f example/ci/pipeline.yaml pipeline.yaml
+cp -rf example/ci/ci/* ci/
+```
+Follow "[Using concourse CI for a CI/CD build](#using-concourse-ci-for-a-cicd-build)" to adapt your pipeline configuartion.
+
```

Comment:
I guess many parts you would write are already present in the ci/cd integration guide readme.

Reply:
removed the copy example